src/main/webapp/app/overview/course-overview.scss

```diff
@@ -195,3 +195,11 @@ canvas#complete-chart {
         width: 100% !important;
     }
 }
+
+.course-exercises {
+    padding-top: 29px;
```

Comment:
Why exactly 29px out of interest?

Reply:
This was following the course information [here](https://github.com/ls1intum/Artemis/blob/310aa64d55c1347b4c2cf6367be551ce1d8f9a4a/src/main/webapp/app/overview/course-overview.scss#L128). 

<img width="1428" alt="Screen Shot 2022-11-02 at 10 28 41 AM" src="https://user-images.githubusercontent.com/10885503/199453933-74b60734-95ca-4fd4-abca-c33dec1c0aea.png">

But I removed it after adding label to the search input.